### PR TITLE
Monitor actions and data source upserts

### PR DESCRIPTION
**Pull request by spolu**
add two monitors:
- Data source upsertion success and failure (with workspaceId visibility)
- Apps Use actions failures (monitor and logs)

### CODE REVIEW

File: front/pages/api/w/[wId]/use/actions/[action]/index.ts [hunk 1]
```diff
@@ -160,24 +172,35 @@ async function handler(
 
       const reader = apiRes.body.getReader();
 
+      const parser = createParser((event) => {
```
Comment (philipperolet) [inline front/pages/api/w/[wId]/use/actions/[action]/index.ts:175]: Humble suggestion on readability: creating a parser + parser.feed in the code makes it hard to understand that it's error checking code (non-standard imho)
quickest suggestion: rename `parser` to `errorDetector` or smth like that
other option: const errorOnAction = fn... then if (errorOnAction(value)) logActionError

**Comment by philipperolet** [comment]
Minor comment otherwise LGTM

File: front/pages/api/w/[wId]/use/actions/[action]/index.ts (hunk 1 above)
Comment (philipperolet) [reply front/pages/api/w/[wId]/use/actions/[action]/index.ts:175]: the idea behind is "understandable quickly on first read"

Comment (spolu) [reply front/pages/api/w/[wId]/use/actions/[action]/index.ts:175]: wilco :+1:

**Review by philipperolet — APPROVED** [review]

**Pull request merged**